app/controllers/users/projects_controller.rb

```diff
@@ -3,4 +3,22 @@ class Users::ProjectsController < ApplicationController
   def new
     @user = current_user
   end
+
+  def create
+    portfolio = User.find(params[:user_id]).portfolio
+    new_project = portfolio.projects.new(project_params)
+    new_project.update(user_id: params[:user_id])
+    if(new_project.save)
+      sleep(1)
+      render js: "/dashboard"
+    else
+      render component: 'ProjectForm', props: { user: current_user}
+    end
+  end
```

Comment:
@neight-allen The sleep was a test experiment. We removed the sleep and it's working as expected. The "render js: "/dashboard"" portion is due to how React works with Rails. Redirecting does not work with axios calls to the server with React. The second rendering is for validations errors so the user stays on the same page with their old inputs